.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -47,26 +55,62 @@ function generateAndroidVersionCode(npmVersion) {
 }
 
 /**
- * Use the react-native-version package to update the version of a native platform.
+ * Find and replace regex in file.
  *
- * @param {String} platform – "ios" or "android"
- * @param {String} versionCode – The version code to update the native platform.
+ * @param {String} filepath
+ * @param {RegExp} findPattern
+ * @param {RegExp} replacePattern
+ * @returns {Promise}
  */
-function updateNativeVersion(platform, versionCode) {
-    if (platform !== 'android' && platform !== 'ios') {
-        console.error('Invalid native platform specified!', platform);
-        core.setFailed();
-        return;
-    }
+function replaceInFile(filepath, findPattern, replacePattern) {
+    return readFileAsync(filepath, {encoding: 'utf-8'})
+        .then(fileContents => fileContents.replace(findPattern, replacePattern))
+        .catch((err) => {
+            console.log(`Failed parsing file ${filepath}`);
+            core.setFailed(err);
+        });
+}
 
-    // Note: We're using `exec` instead of the `react-native-version` javascript sdk to avoid a known issue
-    // with `ncc` and `process.cwd` https://github.com/vercel/webpack-asset-relocator-loader/issues/112
-    exec(`react-native-version --amend --target ${platform} --set-build ${versionCode}`)
+/**
+ * Update the Android app version.
+ *
+ * @param {String} versionName
+ * @param {String} versionCode
+ * @returns {Promise}
+ */
+function updateAndroidVersion(versionName, versionCode) {
+    console.log('Updating android:', `versionName: ${versionName}`, `versionCode: ${versionCode}`);
+    return Promise.all([
+        replaceInFile(BUILD_GRADLE_PATH, /versionName ([0-9.-]*)/, new RegExp(`versionName ${versionName}`)),
+        replaceInFile(BUILD_GRADLE_PATH, /versionCode ([0-9]*)/, new RegExp(`versionCode ${versionCode}`)),
+    ])
         .then(() => {
-            console.log(`Successfully updated ${platform} to ${versionCode}`);
+            console.log('Successfully updated Android!');
         })
         .catch((err) => {
-            console.error('Error updating native version:', `platform: ${platform}`, `versionCode: ${versionCode}`);
+            console.error('Error updating Android');
+            core.setFailed(err);
+        });
+}
+
+/**
+ * Update the iOS app version.
+ *
+ * @param {String} version
+ */
+function updateiOSVersion(version) {
+    console.log(`Updating iOS: ${version}`);
+    Promise.all([
+        exec(`/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString ${version}" ${PLIST_PATH}`),
```

Comment:
I think we want to set this to the version without the build, right? e.g. version = `1.0.1-28` `CFBundleShortVersionString` should be `1.0.1`

Reply:
Updated!